content/docs/plugins/notifications-plugin/_index.md

```diff
@@ -1,70 +1,81 @@
+
+---
+title: "Quick Start"
```

Comment:
there are 2 `title:` elements in this section. pls remove one and change its value to "Notifications Plugin".
otherwise we get this: 
![Selection_264](https://github.com/user-attachments/assets/f1028828-a047-4855-bcd9-3284bb76f2c7)